javaslang/src/main/java/javaslang/collection/Stream.java

```diff
@@ -980,6 +980,17 @@ default int length() {
     }
 
     @Override
+    default Stream<T> leftPadTo(int length, T element) {
+        if (length <= 1) {
```

Comment:
I was trying to avoid using `length()` on Stream as `Stream` is an _lazy_ collection, while calling `length()` on a `Stream` object will trigger a full evaluation.

But looks like both `padTo` and `leftPadTo` have to full evaluate the whole stream any  way, you solution works pretty well.

Shall we make it explicit that these methods should be carefully used on `Stream`?